packages/botonic-core/src/models/events/message/buttons.ts

```diff
@@ -0,0 +1,23 @@
+export interface BaseButton {
+  title: string
```

Comment:
In my opinion naming this attribute `title` is a little bit confusing because it makes me wonder if this is just a title (without being displayed in the bot, just as a description) for the button or is the actual text that will be displayed in the button.
The same thing is happening for the `Reply` interface.

If this attribute is referring to the text displayed in the button, I would consider renaming it to `text`, what do you think?

Reply:
I think `text` is too generic, we want a name that is specific enough to describe "the text that is displayed to the user", for example: `label`, `textLabel`, `display`, `displayText`... However, I still think `title` is good enough. It is used by [Facebook Messenger's API](https://developers.facebook.com/docs/messenger-platform/send-messages/buttons/) (that's where we inherit this name from).

Reply:
You're right, `text` may be too generic, I would go with `label` or `textLabel` then. 
Even if Facebook uses it I think it's not the best naming for this variable, for me `title` represents only the description of the button, not something to be displayed within the Button.

Reply:
leaving it as `title` atm, in the following iterations (integration of parser), I would consider if we change the name to `label`, `textLabel` or similar.